.gitignore

```
@@ -6,3 +6,4 @@ client/server
 .yalc
 *.vsix
 _build
+**/.DS_Store
```

Does this match a top-level `.DS_Store`?

This should be enough
```suggestion
.DS_Store
```